lib/deprecation_toolkit.rb

```diff
@@ -33,5 +33,10 @@ def self.attach_subscriber
   end
 end
 
-require "deprecation_toolkit/minitest_hook"
+if DeprecationToolkit::Configuration.test_runner == :rspec
```

Comment:
I don't think this will work the way we expect to, the configuration from your app won't be loaded yet.

If I take a standard Rails app, the boot process will be to load the gems (thus this file) and then load any configuration file (initializers or whatever). So we can't rely on the configuration, we don't have choice but to do `defined?(RSpec)`